src/backend/web.rs

```diff
@@ -910,8 +910,8 @@ impl crate::Context for Context {
             .collect::<js_sys::Array>();
 
         let mut mapped_desc = web_sys::GpuBindGroupLayoutDescriptor::new(&mapped_bindings);
-        if let Some(label) = desc.label {
-            mapped_desc.label(label);
+        if let Some(label) = &desc.label {
```

Comment:
this looks strange, we should never match on a reference like that. Perhaps, this instead?
```rust
if let Some(label) = desc.label {
  mapped_desc.label(&*label);
}
```

Reply:
That would cause a move error, the choices are basically this:

```rust
if let Some(label) = &desc.label {
  mapped_desc.label(label);
}
```

Or

```rust
if let Some(ref label) = desc.label {
  mapped_desc.label(label);
}
```

Reply:
ok sure `ref label` it is then!